src/complate/bundling.js

```diff
@@ -56,13 +56,15 @@ export default class Bundle {
 		}
 
 		let pending = this._pending = Promise.all([
-			"rodunj",
+			"complate-ast",
 			"rollup",
 			"acorn-jsx"
 		].map(pkg => load(pkg, "complate extension")));
 
-		let [rodunj, rollup, jsx] = await pending;
-		RODUNJ = rodunj.default;
+		let [complate, rollup, jsx] = await pending;
+		COMPLATE = complate.rollupPlugin({
+			prefix: ""
```

Comment:
Code generated by complate-ast depends on a few runtime functions. The assumption is that those are imported directly:

```
import {normalizeAttributes} from "complate-ast/dist/runtime"
```

In order to avoid name collisions, by default there's a prefix:

```
__JSXRuntime__normalizeAttributes({ foo: null })
```

`prefix: ""` disables that prefix. It is only useful if we also enable automatic generation of the import (by default, complate-ast does not emit the import statement).